server/service/src/sync/translations/mod.rs

```diff
@@ -264,8 +282,14 @@ pub(crate) trait SyncTranslation {
     }
 
     // By default matching by change log type
-    fn match_push(&self, row: &ChangelogRow) -> bool {
-        self.change_log_type().as_ref() == Some(&row.table_name)
+    fn match_push(&self, row: &ChangelogRow, r#type: &PushTranslationType) -> bool {
+        match r#type {
+            PushTranslationType::Legacy => self.change_log_type().as_ref() == Some(&row.table_name),
+            // Have to manually specify
+            PushTranslationType::OmSupplyRemoteSitePush => unimplemented!(),
```

Comment:
Is `unimplemented!()` a good idea here? Feels a bit risky, but I can see it's maybe better then returning true or false...